components/Layout.tsx

```diff
@@ -195,31 +195,63 @@ const MobileDocsNav = () => {
-      <DocLink uri='/understanding-json-schema/reference/combining' label='Schema Composition' />
-      <DocLink uri='/understanding-json-schema/reference/conditionals' label='Condition Subschemas' />
-      <DocLink uri='/understanding-json-schema/reference/non_json_data' label='Non-JSON data' />
-      <DocLink uri='/understanding-json-schema/reference/schema' label='Declaring a Dialect' />
+     
+          <SegmentSubtitle label='JSON Schema Reference' />
+          <DocLink uri='/understanding-json-schema/reference/array' label='Type-specific keywords' />
+          <DocLink uri='/understanding-json-schema/reference/string' label='string' />
+          <DocLink uri='/understanding-json-schema/reference/regular_expressions' label='regular expressions' />
+          <DocLink uri='/understanding-json-schema/reference/numeric' label='numeric types' />
+          <DocLink uri='/understanding-json-schema/reference/object' label='object' />
+          <DocLink uri='/understanding-json-schema/reference/array' label='array' />
+          <DocLink uri='/understanding-json-schema/reference/boolean' label='boolean' />
+          <DocLink uri='/understanding-json-schema/reference/null' label='null' />
+          <DocLink uri='/understanding-json-schema/reference/enum' label='Generic keywords' />
+          <DocLink uri='/understanding-json-schema/reference/annotations' label='Media: string-encoding non-JSON data' />
+          <DocLink uri='/understanding-json-schema/reference/combining' label='Schema Composition' />
+          <DocLink uri='/understanding-json-schema/reference/conditionals' label='Applying Subschemas Conditionally' />
+          <DocLink uri='/understanding-json-schema/reference/schema' label='Declaring a Dialect' />
+          <SegmentSubtitle label='JSON Schema Reference' />
+          <DocLink uri='/understanding-json-schema/reference/enum' label='enumerated values' />
+          <DocLink uri='/understanding-json-schema/reference/annotations' label='annotations' />
+          <DocLink uri='/understanding-json-schema/reference/const' label='constant values' />
+          <DocLink uri='/understanding-json-schema/reference/comments' label='comments' />
+        
+          <SegmentSubtitle label='Structuring a complex schema' />
+          <DocLink uri='/understanding-json-schema/reference/enum' label='Schema Identification' />
+          <DocLink uri='/understanding-json-schema/reference/annotations' label='Base URI' />
+          <DocLink uri='/understanding-json-schema/reference/const' label='$ref' />
+          <DocLink uri='/understanding-json-schema/reference/comments' label='$defs' />
+          <DocLink uri='/understanding-json-schema/structuring' label='Recursion' />
+          <DocLink uri='/understanding-json-schema/reference/conditionals' label='Extending Recursive Schemas' />
+          <DocLink uri='/understanding-json-schema/reference/non_json_data' label='Bundling' />
```

Comment:
Please, add top level category at the bottom of the menu called specification linking to "/specification"
<img width="788" alt="Screenshot 2023-04-26 at 20 16 05" src="https://user-images.githubusercontent.com/40007659/234666672-dd71141c-80d0-494b-8417-bd0ab3769899.png">